spec/01-unit/19-hybrid/03-compat_spec.lua

```diff
@@ -342,6 +342,62 @@ describe("kong.clustering.compat", function()
     end)
   end)
 
+
+  for _, strategy in helpers.each_strategy() do
+    local compat_describe = (strategy == "postgres") and describe or pending
+    compat_describe("[#" .. strategy .. "]: check compat for entities those have `updated_at` field", function()
```

Comment:
It should also be tested in the `off` strategy I think?

Reply:
fixed